File: Assignment 4/program2.c
#include<stdio.h>

void FactorRev(int iNo)
{
    int iCnt=0;
    for(iCnt=iNo/2;iCnt>=0;iCnt--)
    {
        if(iNo%iCnt==0)
        {
            printf("%d\t",iCnt);

        }

    }
   
}



int main()
{
    int iValue=0;
 
    printf("Enter the Number:");
    scanf("%d",&iValue);

    FactorRev(iValue);


    return 0;
}

File: Assignment 26/program2.java
import java.util.*;

class Pattern
{ 
    public void Display(int iNO)
    {
        int iCnt=0;
        

        for(iCnt=iNO;iCnt>=1;iCnt--)
        {
            System.out.print(iCnt+ " # ");
        
        }

    }

}

class program2
{
    public static void main(String A[])
    { 
        int iValue=0;

        Scanner sobj=new Scanner(System.in);
        System.out.println("Enetr Number N:");
        iValue=sobj.nextInt();

        Pattern pobj=new Pattern();
        pobj.Display(iValue);

    }
}